ward/fixtures.py

```diff
@@ -129,7 +129,7 @@ def teardown(self):
                 asyncio.get_event_loop().run_until_complete(awaitable)
 
 
-def fixture(func=None, *, scope: Optional[Union[Scope, str]] = Scope.Test):
+def fixture(func=None, *, scope: Union[Scope, str] = Scope.Test):
```

Comment:
Here I'm also a bit unsure. Is there a good reason that `None` should be allowed as input? If yes, then it should probably be handled before `Scope.from_str(scope)`.

Reply:
This shouldn't be `Optional` -- your change here is correct :)